login/login.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    </head>
+    <body>
+        <noscript>To play CardShifter, please either enable JavaScript or use a JavaScript-enabled browser.</noscript>
+        <form name = "login_information">
+            Server:
+            <input name = "server" type = "text" placeholder = "dwarftowers.com" />
```

Comment:
I thought initially there was a field for the port. Not sure that it's actually needed since it will always be 4243 for WebSocket, but just making sure.